tests/autoyast/installation.pm

```diff
@@ -195,6 +195,10 @@ sub run {
             send_key 'ret';    # press enter if grub timeout is disabled, like we have in reinstall scenarios
             last;              # if see grub, we get to the second stage, as it appears after bios-boot which we may miss
         }
+        elsif (match_has_tag('import-untrusted-gpg-key')) {
+            handle_untrusted_gpg_key;
+            next;
```

Comment:
let's `grep` to exclude the needle before `next` to avoid more times matching and lost keys.

Reply:
Done